exporter/clickhouseprofileexporter/ch/access_native_columnar.go

```diff
@@ -53,6 +72,7 @@ func (ch *clickhouseAccessNativeColumnar) InsertBatch(ls plog.Logs) error {
 	timestamp_ns := make([]uint64, sz)
 	typ := make([]string, sz)
 	service_name := make([]string, sz)
+	sample_types_units := make([][]any, sz)
```

Comment:
maybe a special type and not any?

Reply:
here should be any due to clickhouse needs.

Reply:
cant it be like `tags := make([][]tuple, sz)`?

Reply:
tuple have always been resolving to []any in clickhouse golang drivers.
If you can provide examples of another logic, feel free to put an example.

Reply:
I see. You want me to typedef []any to tuple. In spite of it's an unneeded syntactic sugar, I'll try. :)

Reply:
either add the sugar or remove it entirely :)